### PR TITLE
feat: Filtering farming pools & farms

### PR DESCRIPTION
**Pull request by heyimbrucelee**
Filtering farming pools & farms
New filter button 'farming' between active & inactive

### CODE REVIEW

File: src/views/Farms/Farms.tsx [hunk 1]
```diff
@@ -36,7 +36,9 @@ const Farms: React.FC = () => {
 
   const activeFarms = farmsLP.filter((farm) => farm.pid !== 0 && farm.multiplier !== '0X')
   const inactiveFarms = farmsLP.filter((farm) => farm.pid !== 0 && farm.multiplier === '0X')
-
+  const farmingFarms = farmsLP
+    .filter((farm) => farm.pid !== 0 && farm.multiplier !== '0X')
+    .filter((farm) => farm.userData && farm.userData.stakedBalance > new BigNumber(0))
```
Comment (hachiojidev) [inline src/views/Farms/Farms.tsx:41]: No need to for two filters, we can use `activeFarms`

```
const farmingFarms = activeFarms.filter...
```

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Thanks! Just a few things to update

- Left one comment
- Linting errors

**Comment by Chef-Chungus** [comment]
@heyimbrucelee Thanks for taking the time to submit this. We'll award you with a bounty in CAKE after the merge and deployment of this. 

Just some feedback from our designer @ChefBugs

Can you please add this as a toggle switch instead of a tab, for styling information please check our [Figma](https://www.figma.com/file/q5dKq3BuQZ4YzV5pR0Mt2M/Design-System-UI?node-id=2060%3A9400)
image

**Comment by Chef-Chungus** [comment]
@heyimbrucelee come join our slack
Let’s chat about dev, compensation for this and other stuff 
https://join.slack.com/t/cakecommunitydev/shared_invite/zt-l01lbyk3-o47UMVjSps2NEfQivctCZg

File: src/views/Farms/Farms.tsx [hunk 2]
```diff
@@ -34,9 +34,11 @@ const Farms: React.FC = () => {
     }
   }, [account, dispatch, fastRefresh])
 
+  const [stackedOnly,setStackedOnly] = useState(0);
+
   const activeFarms = farmsLP.filter((farm) => farm.pid !== 0 && farm.multiplier !== '0X')
   const inactiveFarms = farmsLP.filter((farm) => farm.pid !== 0 && farm.multiplier === '0X')
-
+  const stackedOnlyFarms = activeFarms.filter((farm) => farm.userData && farm.userData.stakedBalance > new BigNumber(0))
```
Comment (RabbitDoge) [inline src/views/Farms/Farms.tsx:41]: please use https://mikemcl.github.io/bignumber.js/#gt

File: src/views/Farms/Farms.tsx [hunk 3]
```diff
@@ -34,9 +34,11 @@ const Farms: React.FC = () => {
     }
   }, [account, dispatch, fastRefresh])
 
+  const [stackedOnly,setStackedOnly] = useState(0);
```
Comment (RabbitDoge) [inline src/views/Farms/Farms.tsx:37]: should be a bool, not a number

File: src/views/Pools/Syrup.tsx [hunk 4]
```diff
@@ -27,6 +27,8 @@ const Farm: React.FC = () => {
   const pools = usePools(account)
   const bnbPriceUSD = usePriceBnbBusd()
   const block = useBlock()
+  const [stackedOnly,setStackedOnly] = useState(0);
```
Comment (RabbitDoge) [inline src/views/Pools/Syrup.tsx:30]: same

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
this pull request doesn't follow our coding styles.
Please read the contribution guidelines, and setup prettier.

**Review by RabbitDoge — APPROVED** [review]

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**